### PR TITLE
Improve Coverage and add Coveralls

### PR DESCRIPTION
**Pull request by vipulnsward**

### CODE REVIEW

**Comment by vipulnsward** [comment]
cc @neerajdotname @prathamesh-sonpatki

File: Gemfile [hunk 1]
```diff
@@ -69,7 +69,7 @@ gem 'browser'
 gem 'haml-rails'
 
 # intercepts outgoing emails in non-production environment
-gem 'mail_interceptor', github: 'bigbinary/mail_interceptor', group: [:development, :staging]
+gem 'mail_interceptor', github: 'bigbinary/mail_interceptor', group: [:development, :staging, :test]
```
Comment (neerajdotname) [inline Gemfile]: No need to intercept in test. Then in your tests the recipient will change to the intercepted email.

File: test/test_helper.rb [hunk 2]
```diff
@@ -14,7 +17,7 @@ def enable_test_coverage
   end
 end
 
-enable_test_coverage if ENV['COVERAGE']
+enable_local_test_coverage if ENV['COVERAGE']
```
Comment (neerajdotname) [inline test/test_helper.rb]: why the word "local" here ?

**Comment by vipulnsward** [comment]
@neerajdotname updated

**Comment by neerajdotname** [comment]
@vipulnsward can you add to README how to see the coverage report. We use circleci which also can give coverage report. Do we really need this ?

**Comment by vipulnsward** [comment]
I have added badge to README to see coverage.
Coveralls reports are better to view and don't need extra setup on Circle CI

**Comment by prathamesh-sonpatki** [comment]
I will propose adding it in respective projects as and when required. 

Also personally not fan of code coverage tools.

**Comment by vipulnsward** [comment]
Hmm wanted this to report Wheel's Code Coverage, because we keep missing tests sometimes on Wheel.

Will remove Coveralls from this PR.

**Comment by vipulnsward** [comment]
Updated to remove Coveralls.

**Pull request merged**